### PR TITLE
1879 alternate graphic representation fixes

### PR DESCRIPTION
**Pull request by TobiasNx**
@gregorbg also have a look at these changes.

This PR does three things:
1. It fixes the errors created by the `alternateGraphicRepresentation.record.isPartOf` mapping of `4901?`
2. Add the notation to `alternateGraphicRepresentation.record.isPartOf`
3. Add `titleOfSubseries`-Infos to `alternateGraphicRepresentation.record.title`

### CODE REVIEW

**Review by blackwinter — APPROVED** [review]

**Comment by gregorbg** [comment]
Cool, thanks a lot! This alleviates most of my concerns from #1879.

Just one quirk: The test data derived from record 990202474680206441 looks as follows (abridged):

```json
"alternateGraphicRepresentation" : [
  {
    "script" : {
      "id" : "https://unicode.org/iso15924/iso15924.txt#Jpan",
      "label" : "Japanisch (Kanji, Hiragana und Katakana)"
    },
    "record" : {
      "title" : "小原国芳選集, 全人教育論",
      "edition" : [ "第4刷" ],
      "contribution" : [
        {
          "agent" : {
            "label" : "小原, 國芳"
          }
        }
      ]
    }
  }, {
    "script" : {
      "id" : "https://unicode.org/iso15924/iso15924.txt#Kana",
      "label" : "Katakana"
    },
    "record" : {
      "title" : "ゼンジン キョウイクロン"
    }
  }
]
```

where the `Jpan` script has `"title" : "小原国芳選集, 全人教育論"` which reads `Obara Kuniyoshi senshū, zenjin kyōikuron`. The `Kana` script has `"title": "ゼンジン キョウイクロン"` which only reads "Zenjin kyōikuron". I know that this is the fault of the original source data (the main MARCXML source data does not have a `Kana` transliteration for the "Obara Kuniyoshi senshū" part) but it brings up some interesting concerns about how the current `record.title` string is "glued together" somewhat arbitrarily, leading to inconsistencies between the scripts.

File: src/main/resources/alma/fix/macros.fix [hunk 1]
```diff
@@ -40,13 +40,15 @@ do put_macro("alternateGraphicRepresationIsPartOf")
                 end
                 set_array("$AGR.record.isPartOf[].$append.hasSuperordinate[]")                
                 copy_field("$880.a","$AGR.record.$[targetField].label")
```
[the remark this answers is not in the document]
Comment (TobiasNx) [reply src/main/resources/alma/fix/macros.fix:42]: +1 to `isPartOf`

`alternateGraphicRepresationArrayOfObjects` unfortunatly is not that simple:
`"contribution[].$append.agent"`
`"subject[].$append"`

[the remark this answers is not in the document]
Comment (blackwinter) [reply src/main/resources/alma/fix/macros.fix:42]: Why not? `$[targetArray].$append$[targetField]` works with both `targetField: ""` (which could be the default) and `targetField: ".agent"`. But it's up to you...

**Comment by TobiasNx** [comment]
> 

I added a fix that we keep the romanized number if it is only given in romanized numbers.

**Review by dr0i — APPROVED** [review]

**Comment by dr0i** [comment]
Will be deployed next Monday.

**Pull request merged**